CODEOWNERS

```diff
@@ -17,6 +17,7 @@
 /sysdig/internal/client/v2/client.go @filiptubic @mbarbieri @draraksysdig
 /sysdig/internal/client/v2/config.go @filiptubic @mbarbieri @draraksysdig
 /sysdig/internal/client/v2/ibm.go @filiptubic @mbarbieri @draraksysdig
+/sysdig/internal/client/v2/agentaccesskey.go @igoreulalio
```

Comment:
@mbarbieri who is going to be the maintainer of this feature?

Reply:
@IgorEulalio can you add `@filiptubic @mbarbieri @draraksysdig` as maintainers?

Reply:
it's the platform team. We can leave Igor as a gesture of gratitude for working on it 🙂